OpenRA.Mods.Common/Traits/BotModules/HarvesterBotModule.cs

```diff
@@ -13,58 +13,77 @@
 using System.Collections.Generic;
 using System.Linq;
 using OpenRA.Mods.Common.Activities;
+using OpenRA.Mods.Common.AI;
 using OpenRA.Mods.Common.Pathfinder;
-using OpenRA.Mods.Common.Traits;
 using OpenRA.Support;
 using OpenRA.Traits;
 
-namespace OpenRA.Mods.Common.AI
+namespace OpenRA.Mods.Common.Traits
 {
-	class AIHarvesterManager
+	[Desc("Put this on the Player actor. Manages bot harvesters to ensure they always continue harvesting as long as there are resources on the map.")]
+	public class HarvesterBotModuleInfo : ConditionalTraitInfo, IBotModuleInfo
+	{
+		[Desc("Name for identification purposes.")]
```

Comment:
This desc isn't helpful, IMO.
What uses this `Name`? Is it an internal ID (similar to faction etc)?